site/pages/react/authenticate-users.mdx

```diff
@@ -86,6 +86,43 @@ export default function MyLoginPage() {
 }
 ```
 
+### Social Login
```

Comment:
Can you please add an instruction that tells the user they have to go to the Dashboard to enable social auth?

and please explain what effect the fields in the dashboard have on the end product.